### ISSUE TITLE
Call to undefined function GuzzleHttp\Psr7\stream_for()

### ISSUE THREAD

**Issue by Aternus** [cloudinary/cloudinary_php#292]
## Describe the bug in a sentence or two.
Call to undefined function GuzzleHttp\Psr7\stream_for()

## Issue Type (Can be multiple)
- [X] Behaviour - Functions aren’t working as expected (such as generate URL)

## Steps to reproduce
1. Install the latest version of Cloudinary SDK (2.2.0)
2. Attempt to perform the following:
```
try {
    $response = (new CloudinaryUploadApi())->upload(
        'D:\x\y\z\gorilla.png',
        [
            'use_filename'    => true,
            'unique_filename' => false,
            'overwrite'       => true,
            'colors'          => true,
        ]
    );
    Debug::log($response);
} catch (Exception $error) {
    Debug::log($error);
}
```

Expected Result: Upload Successful

Actual Result: 
[02-Jul-2021 23:46:15 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function GuzzleHttp\Psr7\stream_for() in D:\wamp\www\xyz\wp-content\themes\abc\vendor\cloudinary\cloudinary_php\src\Utils\FileUtils.php:168

## Operating System
- [x] All

## Environment and Frameworks (fill in the version numbers)

- PHP Cloudinary SDK version - 2.2.0
- PHP Version - 7.4.21 
- Framework (Laravel, Symphony, etc) - WordPress 5.7.2

**Comment by Aternus** [comment]
Use `Psr7\Utils::streamFor` if you'd like to support Guzzle 7 or specify a lower version of Guzzle

**Comment by kburton-dev** [comment]
I made a pull request to resolve this. You can specify in your composer file to use guzzlehttp/psr7 version 1.8.2, which still has the deprecated method.

**Comment by const-cloudinary** [comment]
@kburton-dev, thank you for contribution!

**Comment by const-cloudinary** [comment]
Fixed in #291

**Comment by const-cloudinary** [comment]
@Aternus , thanks for bringing it to our attention!
